interoperator/controllers/provisioners/sfservicebindingcleaner/sfservicebindingcleaner_controller.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2019 The Service Fabrik Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sfservicebindingcleaner
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	apiErrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/util/retry"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	osbv1alpha1 "github.com/cloudfoundry-incubator/service-fabrik-broker/interoperator/api/osb/v1alpha1"
+)
+
+// ReconcileSFServiceBindingCleaner reconciles a SfServiceBindingCleaner object
+type ReconcileSFServiceBindingCleaner struct {
+	client.Client
+	Log    logr.Logger
+	Scheme *runtime.Scheme
+}
+
+func (r *ReconcileSFServiceBindingCleaner) Reconcile(req ctrl.Request) (ctrl.Result, error) {
+	ctx := context.Background()
+	log := r.Log.WithValues("sfservicebindingcleaner", req.NamespacedName)
+	binding := &osbv1alpha1.SFServiceBinding{}
+	err := r.Get(ctx, req.NamespacedName, binding)
+	if err != nil {
+		if apiErrors.IsNotFound(err) {
+			// Object not found, return. Created objects are automatically
+			// garbage collected. For additional cleanup logic use finalizers.
+			log.Info("binding deleted", "binding", req.NamespacedName.Name)
```

Comment:
`binding_id` is already logged at [#L42](https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/748/files#diff-434b720efc3b854e3eeb0588c721e47eR42) , no need to log it again?

Reply:
Done

Reply:
Done